packages/text-input/src/types.ts

```diff
@@ -3,23 +3,15 @@ export type TextInputAppearanceType =
   | 'neutral'
   | 'minimal'
   | 'underline'
+
 export type TextInputType = 'email' | 'password' | 'tel' | 'hidden' | 'text'
-export interface TextInputProps {
-  id?: string
-  name?: string
+
+export type TextInputProps = React.InputHTMLAttributes<{}> & {
   appearance?: TextInputAppearanceType
-  type?: TextInputType
   borderRadius?: number
-  height?: number
-  width?: number | string
-  required?: boolean
-  className?: string
-  placeholder?: string
-  spellCheck?: boolean
-  defaultValue?: any
-  value?: any
-  disabled?: boolean
-  onChange?: React.ChangeEventHandler<HTMLInputElement>
+  type?: TextInputType
+  /**
+   * Sets visual styling of the text area to be "invalid".
+   */
+  invalid?: boolean
```

Comment:
I suggest adding "full" prop that will automatically set width to 100% - i'm aware that it can be achieved by using `width` prop, but this way developer experience will be consistent with another components (e.g. Button)

Reply:
@Idered ah, and the same goes to `Textarea` component